src/DssSpell.t.base.sol

```diff
@@ -1328,6 +1329,20 @@ contract DssSpellTestBase is Config, DssTest {
         assertEq(vestDai.rxd(_index), _claimed,           "rxd");
     }
 
+    function _checkTransferrableVestMkrAllowance() internal {
+        uint256 vestableAmt;
+
+        for(uint256 i = 1; i <= vestMkr.ids(); i++) {
+            if (vestMkr.valid(i)) {
+                (,,,,,,uint128 tot, uint128 rxd) = vestMkr.awards(i);
+                vestableAmt = vestableAmt + (tot - rxd);
+            }
+        }
+
+        uint256 allowance = gov.allowance(pauseProxy, address(vestMkr));
+        assertGe(allowance, vestableAmt, "TestError/insufficient-mkr-gov-transferrable-vest-allowance");
```

Comment:
Test is called `_checkTransferrableVestMkrAllowance` so I would call this 'insufficient governance transferrable vest mkr allowance" or something.

```suggestion
        assertGe(allowance, vestableAmt, "TestError/insufficient-gov-transferrable-vest-mkr-allowance");
```

Up to you